### PR TITLE
ssh banner: Add tests for ssh banner

### PR DESCRIPTION
**Pull request by TheKharleeci**
bug: 4903

### CODE REVIEW

File: tests/bug-4903-01/test.yaml [hunk 1]
```diff
@@ -0,0 +1,66 @@
+# *** Add configuration here ***
+
```
Comment (jufajardini) [inline tests/bug-4903-01/test.yaml:2]: The fix added by Shivani to this bug isn't yet backported to 5.0.x, that's why we are seeing the CI failures. In order for the test to be run for 6.0.x onwards, can you please add here the option:
```yaml
requires:
  min-version: 6.0
```
Same thing for the other test.yaml file :)

Comment (TheKharleeci) [reply tests/bug-4903-01/test.yaml:2]: Ok. Thank you. I'll do that now.

Comment (jufajardini) [reply tests/bug-4903-01/test.yaml:2]: Perfect, thanks! While you advance with your project and add more s-v tests,  when such failures happen, could you please reach out to us to confirm if this is a case for adding this configuration option or if's something that must be addressed?

**Comment by TheKharleeci** [comment]
Followed by: #618

**Pull request closed without merging**